service-runner/src/main/resources/profiles/default/config.env

```diff
@@ -27,6 +27,7 @@ custody_server.base_url=http://custody-server:8086
 data.type=postgres
 data.server=db:5432
 data.database=postgres
+data.schema=anchor-platform
```

Comment:
This is for default testing configuration and will not impact the backward compatibility.